__init__.py

```diff
@@ -68,94 +69,82 @@ def show_home(self, message):
         """
         self.gui.show_page("Browser.qml", override_idle=120)
 
+    def _file2entry(self, file_url):
+        base, file_extension = os.path.splitext(file_url)
```

Comment:
This does not get the correct extension, which causes all audio files to always load as video, The following fixes the issue:

```
        base = os.path.splitext(file_url)
        file_extension = file_url.split(".")[-1]
```